### DISCUSSION TITLE
Disable auto generating JSON file by default

### DISCUSSION THREAD

**Discussion by Codel1417** [vrchat-community/osc#56] [Ideas]
Having a JSON file for every avatar is just unnecessary for those who aren't using OSC. Maybe disable the generation unless OSC is enabled in game, or provide an option to disable.

Or just not have the config unless explicitly enabled since most will not need it

**Comment by momo-the-monster** [comment]
@Codel1417 this should be the current functionality- is your client creating configs when you have OSC disabled? If so, please file a bug with some details.
